web-components/src/components/combobox/tokens/md-combobox-tokens.js

```diff
@@ -58,17 +58,25 @@ const combobox = {
     },
     item: {
       "hover-bg-color": {
-        light: "#e6f9fc",
-        dark: colors.theme[90].name
+        light: colors.blue[20].name,
+        dark: colors.blue[50].name
       },
       "focus-bg-color": {
-        light: "#b8f2ff",
-        dark: colors.theme[60].name
+        light: colors.blue[70].name,
+        dark: colors.blue[40].name
       },
       "select-bg-color": {
         light: colors.white[60].name,
         dark: colors.gray[90].name
       },
+      "select-font-color": {
+        light: colors.gray[100].name,
+        dark: colors.gray[90].name
+      },
+      "selected-bg-color": {
+        light: colors.blue[50].name,
+        dark: colors.blue[40].name
+      },
       "checked-bg-color": {
         light: colors.theme[50].name,
         dark: colors.theme[60].name
```

Comment:
If you are changing lm token change same in md token

Reply:
reverted the change to original
checked-bg-color